charts/datadoc/values.schema.json

```
@@ -17,6 +17,9 @@
               "description": "Aktiver datatilgang",
               "default": true
             }
+          },
+          "x-onyxia": {
+            "hidden": false
```

```suggestion
            "hidden": true
```

It should be hidden right?